src/chainparams.cpp

```diff
@@ -377,11 +383,13 @@ class CRegTestParams : public CChainParams {
         consensus.signet_challenge.clear();
         consensus.nSubsidyHalvingInterval = 150;
         consensus.BIP16Exception = uint256();
-        consensus.BIP34Height = 500; // BIP34 activated on regtest (Used in functional tests)
+        // Dogecoin: Raise this a long way so it's below 3 * COINBASE_MATURITY still
+        consensus.BIP34Height = 1000; // BIP34 activated on regtest (Used in functional tests)
         consensus.BIP34Hash = uint256();
         consensus.BIP65Height = 1351; // BIP65 activated on regtest (Used in functional tests)
         consensus.BIP66Height = 1251; // BIP66 activated on regtest (Used in functional tests)
-        consensus.CSVHeight = 432; // CSV activated on regtest (Used in rpc activation tests)
+        // Dogecoin: Note this needs to be a multiple of nMinerConfirmationWindow
+        consensus.CSVHeight = 720; // CSV activated on regtest (Used in rpc activation tests)
         consensus.SegwitHeight = 0; // SEGWIT is always activated on regtest unless overridden
```

Comment:
Sure, mind if it's a later PR?

Reply:
My thinking was that if we set each unimplemented fork to something like `INT_MAX` in an atomic commit, then that commit can be reversed, when implementing the soft fork supermajority, which then can be reversed when implementing post-fork fixation, and then in subsequent ports like we're doing right now, we can just ignore all the dummy and reversals, and cherry-pick the final change from the fixation.

Reply:
Okay. So. Main thing is, why do we care that much about RegTest values, and... if we change this, we'll break a lot of tests. Is the intent they're disabled or ignored broken? We can't fix them (because we can never test SegWit works, for example)?

Reply:
I care about them because I would like to assure that we don't have a regression against current protocol implementation without this, and then subsequently we can test any new protocol activations later. The reason why I point to it now is ONLY because we can potentially have a clean commit path that simplifies future releases rather than having to do it manually each time. This is of course only an optimization in process, not in code.

Reply:
Created https://github.com/dogecoin/dogecoin/issues/1831 to track this, as the first draft killed around 70 of the functional tests.